### ISSUE TITLE
WSJ exp command

### ISSUE THREAD

**Issue by Entonytang** [rizar/attention-lvcsr#3]
$LVSR/exp/wsj/decode.sh wsj_paper6 <beam-size>
in decode.sh, three parameters should be given.....while only 2 parameters appear in command.

**Comment by rizar** [comment]
Right, there are two more. We will fix it at some point, but for now,
please read the script, it is not very long.

**Comment by Entonytang** [comment]
I know. I use the command:
$LVSR/exp/wsj/decode.sh wsj_paper6 valid 200...... 
and I hope you can give some descriptions about the config file like wsj_paper.yaml......
